pyomo/contrib/trustregion/TRF.py

```diff
@@ -2,18 +2,29 @@
 #
 #  Pyomo: Python Optimization Modeling Objects
 #  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
-#  Under the terms of Contract DE-NA0003525 with National Technology and 
-#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain 
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
 #  rights in this software.
+#
+#  The Institute for the Design of Advanced Energy Systems Integrated Platform
+#  Framework (IDAES IP) was produced under the DOE Institute for the
```

Comment:
As this was not migrated over from IDAES-PSE, I don't think this falls under the IDAES-PSE copyright assertion.  Instead, this is "IDAES Generated IP", which falls to the individual M&O contractor to assert copyright (so it still falls under the original copyright).  I think the correct thing to do here is:

```
#  Development of this module was conducted as part of the Institute for
#  the Design of Advanced Energy Systems (IDAES) with support through the
#  Simulation-Based Engineering, Crosscutting Research Program within the
#  U.S. Department of Energy’s Office of Fossil Energy and Carbon Management.
```